### PR TITLE
Gurobi doc

### PR DESCRIPTION
**Pull request by Sparrow0hawk**
adding in documentation for the Gurobi module and instructions for using with R, Python and MATLAB

### CODE REVIEW

File: book/software/gurobi.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+## Gurobi module
+
+To access the Gurobi solver you need to load the `gurobi` module. You can do this with the command:
+
+```bash
+$ module add test gurobi
+```
+
+```note
```
Comment (jhodrien) [inline book/software/gurobi.md:13]: Throws a build warning.  Used {note} elsewhere?

File: book/software/gurobi.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+# installing with conda
+$ conda install -c gurobi gurobi
+```
+
+You can then use the Gurobi Python API by adding the line `import gurobi` to your python script and calling functions from the module.
+
+### R setup
+
+```note
```
Comment (jhodrien) [inline book/software/gurobi.md:41]: Throws a build warning.  Used {note} elsewhere?

**Review by jhodrien — CHANGES_REQUESTED** [review]
Needs a _toc.yml entry.

**Comment by Sparrow0hawk** [comment]
@jhodrien should have solved these in latest push

**Review by jhodrien — APPROVED** [review]

**Pull request merged**